APMrover2/Log.cpp

```diff
@@ -112,8 +112,8 @@ void Rover::Log_Write_Nav_Tuning()
         LOG_PACKET_HEADER_INIT(LOG_NTUN_MSG),
         time_us             : AP_HAL::micros64(),
         wp_distance         : control_mode->get_distance_to_destination(),
-        wp_bearing_cd       : (uint16_t)wrap_360_cd(control_mode->wp_bearing() * 100),
-        nav_bearing_cd      : (uint16_t)wrap_360_cd(control_mode->nav_bearing() * 100),
+        wp_bearing          : wrap_360(control_mode->wp_bearing()),
```

Comment:
Don't need the `wrap_360` here.  If it's not between 0 and 360 we have a problem :-)